### PR TITLE
Tanium - support multiple Tanium Trace capabilities

### PR DESCRIPTION
**Pull request by shaniacht1**
'tn-trace-get-live-connections'
'tn-trace-connect'
'tn-trace-capture-snapshot'
'tn-trace-list-snapshots'
'tn-trace-download-snapshot'
'tn-trace-get-evidence-files-details'
'tn-trace-get-file'  (requested by Fedex)

### CODE REVIEW

File: Integrations/integration-Tanium.yml [hunk 1]
```diff
@@ -1122,4 +1292,167 @@ script:
...
+    - contextPath: File.SHA1
+      description: SHA1 hash of the file
+      type: string
+    - contextPath: File.EntryID
+      description: Demisto entry ID of the file
+      type: string
+    - contextPath: File.SSDeep
+      description: SSDeep hash of the file
+      type: string
+  - name: tn-trace-capture-snapshot
```
[the remark this answers is not in the document]
Comment (shaniacht1) [reply Integrations/integration-Tanium.yml]: don't think so. If you had them to run the command, you'll have them to get the snapshot

File: Integrations/integration-Tanium.yml [hunk 2]
```diff
@@ -1122,4 +1292,167 @@ script:
...
+    description: Creates a local or remote connection
+  - name: tn-trace-get-evidence-files-details
+    arguments:
+    - name: offset
+      description: Offset into the result list
+      defaultValue: "0"
+    - name: limit
+      description: Maximum number of results to return
+      defaultValue: "25"
+    outputs:
```
[the remark this answers is not in the document]
Comment (shaniacht1) [reply Integrations/integration-Tanium.yml]: no. it doesn't return actual files. just data.

File: Integrations/integration-Tanium.yml [hunk 3]
```diff
@@ -1122,4 +1292,167 @@ script:
...
+      description: Name of host where the file is located
+      type: string
+    - contextPath: Tanium.EvidenceFiles.created
+      description: File creation date
+      type: date
+    - contextPath: Tanium.EvidenceFiles.created_by
+      description: Name of entity that created the file
+      type: string
+    - contextPath: Tanium.EvidenceFiles.hash
+      description: File hash
```
[the remark this answers is not in the document]
Comment (shaniacht1) [reply Integrations/integration-Tanium.yml]: no

File: Integrations/integration-Tanium.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (shaniacht1) [reply Integrations/integration-Tanium.yml]: no. cause you had them to run the command, so you'll have them later as well

**Comment by dorsha** [comment]
@shaniacht1 / @Itay4 any update here?

**Pull request closed without merging**